### ISSUE TITLE
whoami -l returns "Available storage:   0.00  Bytes"

### ISSUE THREAD

**Issue by r1n0x** [meganz/MEGAcmd#676]
> OS: Windows 10 21H1

Hello,
I'm currently trying to make an app for uploading (ultimately supposed to be an non-interactive mode, handled by python) 
and I would like to to check if account has enough available space . (to check if upload is even possible)

I've wanted to use whoami -l to get my account available storage but MEGAcmd seems not to like my idea.
(Idk what log -c / -s really does but output is still the same, I tried killing megacmdserver too) 

I get same output on these types of accounts (cmd and powershell tested):
- 2TB MEGA PRO ACCOUNT (1TB+ available space)
- 50GB MEGA ACCOUNT  (10GB+ available space)
- 20GB MEGA ACCOUNT (700MB, even after uploading and disk quota going through the roof,  output is still same as on screenshot)

> Screenshot, taken on 2TB PRO account, tested across mutliple IPs (Cloudflare WARP / ISP) and with reinstalling MEGAcmd
![image](https://user-images.githubusercontent.com/16156772/173257729-0a80aab9-99ab-4228-aca7-2f6411b7539a.png)

Do you guys have any fixes or another idea how I could get available storage? (preferably bytes)

**Comment by r1n0x** [comment]
I decided to use "df" (it even gives bytes as response) instead of "whami -l"

![image](https://user-images.githubusercontent.com/16156772/173446803-c4cac558-2bc1-42bf-95aa-d828ed1672ad.png)

**Comment by ViperCode7** [comment]
That's fine, but the needed info was about free space. As it happens, I too need that value.
Where can I get the value of available free space in a command?
As it is, I'm on Windows, I have to resort to powershell to process that df output and get the free space.

**Comment by r1n0x** [comment]
@ViperCode7 
Hey, 
I don't really remember, but from what can I see you could just substract used storage from all storage (3 line from the bottom).
53687091200 - 40346002970 = 13341088230
By using that you know that you have 13341088230 bytes free on a drive.

**Comment by ViperCode7** [comment]
Yes, that I already did, but since I'm on Windows, cmd runs with 32bit ints so ops with such long numbers require workarounds. I've opted for PowerShell to do this but still, would've been better for a switch in df that would output something pickable by a for statement, a structured format instead of a human readable format.
